server/integration-test/src/test/java/com/oceanbase/odc/service/userrecentbrowsing/UserRecentBrowsingServiceTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.userrecentbrowsing;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import com.oceanbase.odc.metadb.userrecentbrowsing.UserRecentBrowsingEntity;
+import com.oceanbase.odc.metadb.userrecentbrowsing.UserRecentBrowsingRepository;
+import com.oceanbase.odc.server.OdcServer;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = OdcServer.class)
+@EnableTransactionManagement
+public class UserRecentBrowsingServiceTest {
+
+    @Autowired
+    private UserRecentBrowsingService userRecentBrowsingService;
+
+    @Autowired
+    private UserRecentBrowsingRepository userRecentBrowsingRepository;
+
+    private Long projectId;
+    private Long userId;
+    private Long itemId;
+    private BrowseItemType itemType;
+    private Long browseMills;
+
+    @Before
+    public void setUp() {
+        projectId = System.currentTimeMillis();
+        userId = 1L;
+        itemId = 1L;
+        itemType = BrowseItemType.WORKSHEET;
+        browseMills = System.currentTimeMillis();
+    }
+
+    @After
+    public void tearDown() {
+        userRecentBrowsingRepository.delete(UserRecentBrowsingEntity.builder()
```

Comment:
delete should also process in setUp, will make test logic more stable

Reply:
Okay，I'll make this change.